links/links_test.go

```diff
@@ -95,35 +126,20 @@ func TestGetLinksRequest(t *testing.T) {
 	}))
 	defer ts.Close()
 
-	client.linksEndPoint = ts.URL
+	client.linksEndpoint = ts.URL
 
-	_, err := client.LinkStats(context.Background(), "https://mock", StatOptions{DurationDays: 7})
+	ls, err := client.LinkStats(context.Background(), "https://mock", StatOptions{LastNDays: 7})
 	if err != nil {
 		t.Fatal(err)
 	}
 
 	wantRequestURI := "/https%3A%2F%2Fmock/linkStats?durationDays=7"
 	if tr.RequestURI != wantRequestURI {
-		t.Errorf("expecting RequestURI: %q, got %q", tr.RequestURI, wantRequestURI)
-	}
-}
-func TestGetLinksStats(t *testing.T) {
-	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		w.Header().Set("Content-Type", "application/json")
-		w.Write([]byte(testLinkStatsResponse))
-	}))
-	defer ts.Close()
-
-	client.linksEndPoint = ts.URL
-
-	ls, err := client.LinkStats(context.Background(), "https://mock", StatOptions{DurationDays: 7})
-	if err != nil {
-		t.Fatal(err)
+		t.Errorf("RequestURI = %q; want %q", tr.RequestURI, wantRequestURI)
 	}
-	wantLinkStats := getWantedStatResult()
 
-	if !reflect.DeepEqual(*ls, wantLinkStats) {
-		t.Errorf("read json file, got %#v, want: %#v", *ls, wantLinkStats)
+	if !reflect.DeepEqual(ls, wantedStatResult) {
+		t.Errorf("read json file, got %#v; want: %#v", ls, wantedStatResult)
```

Comment:
Please be consistent with other test packages when formatting error messages. Tis should be:

`LinkStats() = %#v; want = %#v`